runtimes/php/php-apps.md

```diff
@@ -447,6 +447,15 @@ Laravel doesn't need Monolog to retrieve logs via Clever console or Clever CLI.
 
 Then, set `APP_LOG=syslog` as Clever application environment variable.
 
+## Timezone configuration
+
+All instances on Clever Cloud run on the UTC timezone. We recommend to handle all your dates in UTC internally, and only handle timezones when reading or displaying dates. You can set PHP's time zone setting with `.user.ini`.
```

Comment:
I would put "You can set...." in another paragraph and add something like 'However,".

Reply:
done, not sure about the "however", they are not opposed imo